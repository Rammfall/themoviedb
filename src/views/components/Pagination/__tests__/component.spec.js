import React from 'react'
import { shallow } from 'enzyme'

import PaginationComponent from '../component'

describe('PaginationComponent', () => {
  it('matches snapshot', () => {
    const component = shallow(<PaginationComponent onChange={jest.fn} />)

    expect(component).toMatchSnapshot()
  })
})
