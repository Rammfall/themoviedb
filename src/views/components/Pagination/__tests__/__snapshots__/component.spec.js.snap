// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationComponent matches snapshot 1`] = `
<Row
  justify="center"
  type="flex"
>
  <Col>
    <Pagination
      className="pagination"
      current={1}
      defaultCurrent={1}
      defaultPageSize={20}
      onChange={[Function]}
      showSizeChanger={false}
      total={1}
    />
  </Col>
</Row>
`;
